packages/testing/src/execution_testing/cli/tox_helpers.py

```diff
@@ -210,7 +226,8 @@ def validate_changelog() -> None:
     - A period (.) for regular entries
     - A colon (:) for section headers that introduce lists
     """
-    changelog_path = Path("docs/CHANGELOG.md")
+    project_root = find_project_root()
+    changelog_path = Path(project_root / "docs/CHANGELOG.md")
```

Comment:
`uv run markdownlintcli2_soft_fail` only works from the project root (or it won't find its config file `.markdownlint.yaml`, so we should rather just check that a `pyproject.toml` is present in the cwd and give an appropriate error message if not.